src/OpenLoco/src/Localisation/Conversion.cpp

```diff
@@ -88,6 +215,17 @@ namespace OpenLoco::Localisation
             return it->locoCode;
         }
 
+        const auto it2 = std::ranges::lower_bound(
```

Comment:
can we name this better than `it2`? reading the code its the same iteration for both tables - potentially move to a common method and it can just be a single `it`?